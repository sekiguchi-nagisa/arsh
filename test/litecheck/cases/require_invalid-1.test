#!/bin/sh
echo 12345
echo ABCDE
echo abcde
echo @@@@@
# REQUIRE: command ()
# CHECK: 12345
# CHECK: ABCDE
# CHECK: abcde
# CHECK: @@@@@