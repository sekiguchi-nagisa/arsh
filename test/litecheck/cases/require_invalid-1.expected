require_invalid-1.test:6: [error] invalid REQUIRE directive: `command ()', caused by
[syntax error] expected `{'
 --> (loaded):1:11
command ()
          ^

